circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/hive/RenameTableOperation.java

```diff
@@ -40,20 +40,27 @@ public RenameTableOperation(DropTableService dropTableService) {
    * renamed.
    */
   public void execute(CloseableMetaStoreClient client, Table from, Table to) throws Exception {
-    LOG
-        .info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(),
-            to.getTableName());
+    LOG.info("Renaming table {}.{} to {}.{}", from.getDbName(), from.getTableName(), to.getDbName(), to.getTableName());
     Table fromTable = client.getTable(from.getDbName(), from.getTableName());
     Table toTable = client.getTable(to.getDbName(), to.getTableName());
+    String fromDatabaseName = fromTable.getDbName();
     String fromTableName = fromTable.getTableName();
     String toTableName = toTable.getTableName();
     String toDelete = toTableName + DELETE_ME;
     try {
+      fromTable.setDbName(toTable.getDbName());
       fromTable.setTableName(toTableName);
       toTable.setTableName(toDelete);
+      LOG
```

Comment:
Might be worth adding in the comment lines above the logs like we have in Cloverleaf? https://github.expedia.biz/eg-data-lake/cloverleaf/blob/5c948f54c72558260a6744a9d4a752882d61ea99/cloverleaf-steps/src/main/java/com/hotels/bdp/cloverleaf/hive/table/RenameTableOperation.java I find it a bit easier to follow

Reply:
ok